evervault.go

```diff
@@ -333,8 +334,8 @@ func (c *Client) decryptToString(encryptedData string) (string, error) {
 	}
 
 	decryptedString, ok := decryptResponse.(string)
-	if !ok {
-		return "", ErrInvalidDataType
+	if ok {
+		return decryptedString, nil
```

Comment:
this removes the error path entirely, is this intended?

Reply:
reading with more context from the file - is this to support the byte array code path? If so, should probably rename the func to make that more obvious

Reply:
No this was just me being stupid. This function isn't needed anymore. The decrypt function itself converts everything to a string and each type function handles from there.